.github/workflows/gh_registry.yml

```diff
@@ -1,21 +1,50 @@
 on:
   workflow_dispatch:
-    inputs:
-      image:
-        type: string
-        required: true
-        description: The full name of the image, including tag.
-      platforms:
-        type: string
-        required: false
-        description: The platform(s) for which the image should be available. 
-        default: linux/amd64,linux/arm64
-  
-name: Push to GHCR
+  schedule:
```

Comment:
1) I didn't hardcode the repo owner anywhere, so packages should actually be pushed to your own org as part of this workflow.
2) You can always disable a specific workflow on your repo. Overall, it hence seems more friendly to allow this workflow to be run on forks depending on whether that repo enabled it, rather than having to have an additional commit on forks to enable the workflow if desired.

Reply:
> 1. I didn't hardcode the repo owner anywhere, so packages should actually be pushed to your own org as part of this workflow.

Does that mean they will be pushed to ghcr.io/bmhowe23 for me? Or would it auto-detect that I am part of the NVIDIA org and push it to NVIDIA's folder?

> 2. You can always disable a specific workflow on your repo.

True, but I suspect most of the people who fork our repo don't take the time to do that. So I think we'd end up with ~92 forks all doing the same thing every night and making separate copies of the same data. Is there a way to make it default to off for everybody else and just have the NVIDIA repo enable it?